Player/Player.gd

```diff
@@ -7,6 +7,8 @@ onready var signals = get_node("/root/Signals")
 
 var is_dead: bool = false
 
+var fallHeight = 4
```

Comment:
Unused variable that should be used. It is probably best to raise this variable to something like 10 or 15 so player movement isn't restricted too much.